dotcom-rendering/src/client/ophan/ophan.ts

```diff
@@ -13,6 +13,47 @@ export type OphanRecordFunction = (
 	callback?: () => void,
 ) => void;
 
+/**
+ * Loads Ophan (if it hasn't already been loaded) and returns a promise of Ophan's methods.
+ */
+export const getOphan = async (): Promise<
+	NonNullable<typeof window.guardian.ophan>
+> => {
+	// @ts-expect-error -- side effect only
+	await import(/* webpackMode: "eager" */ 'ophan-tracker-js');
```

Comment:
this moves the place that ophan is added to the page from a side effect of evaluating the entry chunk to the first time `getOphan` is called. 

i think this is ok? 

it doesn't delay the time needed to send the first successful record in ophan (it's still (eval_ophan + send_record), and when used everywhere will make it impossible to try to do that before ophan is loaded.